cfgov/v1/models/organisms.py

```diff
@@ -13,6 +15,48 @@ class Meta:
         template = '_includes/organisms/well.html'
 
 
+class FullWidthText(blocks.StructBlock):
+    content = blocks.RichTextBlock(required=True)
+
+    class Meta:
+        icon = 'title'
+        template = '_includes/organisms/full-width-text.html'
+
+
+class PostPreview(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(required=True)
+    authors = blocks.ListBlock(blocks.CharBlock(required=False))
+    published_date = blocks.DateTimeBlock(required=True)
+
+    image = atoms.ImageBasic(required=False)
+
+    post_category_1 = ref.IconCategoryChoiceBlock(required=True)
+
+    post_category_2 = ref.IconCategoryChoiceBlock(required=False)
+
+    post_tags = blocks.ListBlock(blocks.CharBlock(required=False))
+
+    event = blocks.StructBlock([
+        ('start_date', blocks.CharBlock(required=False)),
```

Comment:
@KimberlyMunoz i think event start date being a string might confuse content editors. Can you still your date magic 
```
<time class="calendar-icon"
                      data-month="{{ value.event.start_date | date('%b') }}"
                     data-day="{{ value.event.start_date | date('%-d') }}"
                     datetime="{{ value.event.start_date | date('%c') }}">
```
if its a date field like the `published_date`